### PR TITLE
Update readme file

### PR DESCRIPTION
**Pull request by kazimdsaidul**
## Why? (ticket link or issue description)
Give an instruction  to run the project by command link 

## What was done and how?
Dev: `flutter run --flavor dev lib/main_dev.dart`
Prod: `flutter run --flavor dev lib/main_prod.dart`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,6 +23,11 @@ for data and observe the response. Till then it may display a loader to the user
 # To configure and run this project [check here](readme_configuration_guideline.md) 
 You will find at above link step by step instructions with screenshots.
 
+# run this project by command line
+Dev: `flutter run --flavor dev lib/main_dev.dart`
+
+Prod: `flutter run --flavor dev lib/main_prod.dart`
```
Comment (hasancse91) [inline README.md:29]: Should it be `flutter run --flavor prod lib/main_prod.dart`?
I found `--flavor dev` here.

**Review by hasancse91 — CHANGES_REQUESTED** [review]
Please check the comment.

**Comment by kazimdsaidul** [comment]
Sorry, i mistake. I change it

**Review by hasancse91 — APPROVED** [review]
Thank you for your contribution.

**Pull request merged**